vscode-lean4/src/abbreviation/abbreviations.json

```diff
@@ -1361,6 +1381,7 @@
     "P0": "Π₀",
     "Pi_0": "Π₀",
     "P_0": "Π₀",
+    "directsum": "Π₀",
```

Comment:
This one is at odds with mathlib, which uses a different symbol for direct sums.

Reply:
Oops, I copied from the definition of `DirectSum` instead of the notation line.